products/dotnet/get-started.adoc

```diff
@@ -35,17 +35,21 @@ Finally, yes, you can install on bare metal if you wish. #{site.base_url}/produc
 
 Here’s the list of instructions to install .NET Core on Red Hat Enterprise Linux:
 
-1. Run `sudo yum install -y rh-dotnetcore10`
+1. Register your RHEL machine using your developer subscription
+2. Run `subscription-manager repos --enable=rhel-7-server-dotnet-rpms`
```

Comment:
Some suggestions for this section:
* Take a look at the SCL get started guides.  They cover the same topics but use the SCL repo instead of the dotnet repo.
* Registering the rhel installation isn't a "step".   It's a prerequisite.  If they need help registering they should be pointed to /products/rhel/get-started
* subscription manager needs root.  Subscription Manager GUI will prompt for root. I don't remember off the top of my head if the CLI version will prompt.
* sudo does not work out of the box on many RHEL installations.  When I've tested a vanilla rhel-7-server install (like you get with dev. suite.) sudo fails unless sudoers has been changed, or the user has been added to group wheel.

So I would instruct them to
1. become root with su,
2. run subscription-manager,
3. yum install 

Again feel free to borrow any of the text form /products/softwarecollecitons/get-started-*.adoc.